app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = <%= t 'Ruby on Rails Tutorial Sample App' %>
```

Comment:
```suggestion
    base_title = t 'Ruby on Rails Tutorial Sample App'
```

Reply:
if it using i18n, using the bottom code is correct method?